README.md

```diff
@@ -286,6 +290,7 @@ Further details on environments on https://docs.openstack.org/heat/latest/templa
 ---
 parameters:
   availability_zone: south-1
+  volume_az: south-1
```

Comment:
Can we use volume_availability_zone as parameter name? Then it is consistent with the availability_zone parameter.

Reply:
Sure, no problem.
With the logic of your parameter naming, it would probably be availability_zone_volume.